ios/Podfile.lock

```diff
@@ -1907,13 +1907,13 @@ SPEC CHECKSUMS:
   RNSVG: d787d64ca06b9158e763ad2638a8c4edce00782a
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
   SwiftProtobuf: 7773c4e96a99d7b8ab7cda0fc30a883732ff93b1
-  TimeReactNativeLib: 10b8725d54862724ac774207128803d0474ed8f0
+  TimeReactNativeLib: 45adee64d1f0d97fd321f6775552f5cd5a7fe316
   TokenCore: c66fccf39c10238ff753b28956134c489d7ff0dd
-  TokenStateReactNativeLib: 0f6875d9d039385a07456588701f638dc5463a79
+  TokenStateReactNativeLib: ee91eddc1159d7efab18960b8afd90fdfbb6f77c
   Turf: 13d1a92d969ca0311bbc26e8356cca178ce95da2
   Yoga: 3efc43e0d48686ce2e8c60f99d4e6bd349aff981
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
 PODFILE CHECKSUM: 65771909d1de0011499a623efd21303801625f9f
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.1
```

Comment:
The changes in this file are probably not intended to be included?

Reply:
We should keep 1.12.1 because that is what we have defined on the gemfiles, I'm going to create a task to upgrade to the newest version.